### ISSUE TITLE
Attributes are dismissed in config file

### ISSUE THREAD

**Issue by Robot-DaneelOlivaw** [v2ray/domain-list-community#585]
As per https://github.com/v2ray/domain-list-community#attributes, I tried [`geosite:baidu@ads`](https://github.com/v2ray/domain-list-community/blob/master/data/baidu-ads), [`geosite:google@ads`](https://github.com/v2ray/domain-list-community/blob/master/data/google-ads) and [`geosite:steam@cdn`](https://github.com/v2ray/domain-list-community/blob/master/data/steam) in config file:
```
{
  "type": "field",
  "domain": [
    "geosite:baidu@ads",
    "geosite:google@ads"
  ],
  "outboundTag": "blocked"
}
```
or
```
{
  "type": "field",
  "domain": [
    "geosite:steam@cdn"
  ],
  "outboundTag": "direct"
}
```
V2Ray returned `v2ray.com/core/app/router: this rule has no effective fields`. All of them were dismissed, not even taken as `geosite:baidu` or `geosite:steam` (`geosite:google-ads` hasn't been tagged). Although `geosite:baidu-ads` and `geosite:google-ads`still worked. I wonder whether this function has been deployed or am I just doing it wrong?

**Comment by Loyalsoldier** [comment]
Neither `google` nor `baidu` has `@ads` attr inside. Don't work definitely.

**Comment by EpLiar** [comment]
> V2Ray returned v2ray.com/core/app/router: this rule has no effective fields.

@Loyalsoldier In fact, I test with `geosite:adcolony-ads@ads`, core return a same error.

[adcolony-ads](https://github.com/v2ray/domain-list-community/blob/master/data/adcolony-ads)' content:

```
adcolony.com @ads
```

**Comment by LamJH** [comment]
> Neither `google` nor `baidu` has `@ads` attr inside. Don't work definitely. The one(google) in README just an example.

You're right. But they do exist in `baidu-ads` which is included in `baidu`.

> I test with `geosite:adcolony-ads@ads`, core return a same error.

Maybe this function has been deployed at all.

**Comment by Loyalsoldier** [comment]
Don't know if it is a bug for now. I would like to read the source code of V2Ray's router package, don't know whether I could figure it out. Or you can file an issue in `v2ray-core` project to draw attentions to help you with this.

**Comment by EpLiar** [comment]
@Loyalsoldier I found a commit that describes the syntax of this README and the date of submission is December 8, 2018. 

See https://github.com/v2ray/domain-list-community/commit/8b28f90a8ffc6c67302f08b45659c73600959a54.

If you are going to look at the source code of the `router`, this date may be of some help to you.

**Comment by Robot-DaneelOlivaw** [comment]
Close. Fixed in v2fly/domain-list-community#79.
